js/data.js

```diff
@@ -849,6 +1003,17 @@ var data = {
         dozent: ["Dr. Matthias Uflacker", "Dipl.-Kfm. Jens Krüger", "Thomas Kowark", "Franziska Häger"],
         nameLV: "Global Team-Based Product Innovation & Engineering III - ME 310 Coaching Research"
     },
+    programmierungVerteilterSysteme: {
+        kurz: "ParProg",
```

Comment:
Yes, you're right.
Guess we should still change the old 6LP ParProg title from "Parallel<br />Programming" to "ParProg 6LP", for consistency reasons. I'll do that.
Would still name this "ParProg 3LP".